src/EnergyPlus/DXCoils.cc

```diff
@@ -12189,8 +12198,10 @@ Real64 ValidateADP(EnergyPlusData &state,
                     SHR += 0.001; // increase SHR slowly until ADP temps are resolved
                 }
             }
-            if (SHR > 0.8)
-                bStillValidating = false; // have to stop somewhere, this is the upper limit of SHR empirical model (see Autosizing/CoolingSHRSizing)
+            if (SHR > shrADPMax) {
+                SHR = shrADPMax;
+                bStillValidating = false;
```

Comment:
This all looks good. I think it would be a good idea to report a severe that describes why the user inputs, or I guess autosized inputs, cause the outlet condition to exceed the saturation curve. A warning the shows the supply air T, w and the DP at saturation(w) that shows the outlet T is left of the saturation curve. At least then the user will know what caused the design failure and a warning will be available to help correct the initial problem. This input issue might be happening more than we know.

Reply:
Sure! I'll push some changes.